workshops/geometric_pattern/README.md

```diff
@@ -0,0 +1,538 @@
+# Geometric Pattern
+
+We'll be creating a cool graphic like the one above, using JavaScript and p5.js.
+[p5.js](http://p5js.org/) is a library for making stuff in conjunction with the HTML canvas element. It is nothing more than JS code written to make commonly-desired functionality much more accessible. It is a JS port of Processing, a software for making pictures and visuals with code. 
+
+## Part I: Set-up
+
+First, go to Cloud9 and open up your `projects` workspace by pressing Open.
+
+![](img/c9_dash.png)
+
+Once in the workspace, right-click the `projects` folder on the left and select `New Folder`. Name it `geometric_pattern`.
+
+Next, right-click the `geometric_pattern` folder, select `New File`, and name it `index.html`.
+Then, right-click the `geometric_pattern` folder again, select `New File` again, and this time, name it `main.js`.
+
+## Part II: Priming the Files
+
+### Priming the HTML File
+
+Just as we've done previously, we'll be typing this base structure into our HTML file:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
+  <body>
+  </body>
+</html>
+```
+
+Now, let's add our JavaScript dependencies. This time around, we'll be using a library called p5.js. Let's add that into our **body** with a `<script>` tag:
```

Comment:
We've mentioned dependencies in Soundboard, which comes before this. But I'll add in an extra note about it.